30-indoor-air-quality-and-gas-leak/firmware/.vscode/c_cpp_properties.json

```diff
@@ -1,34 +1,137 @@
+//
+// !!! WARNING !!! AUTO-GENERATED FILE!
+// PLEASE DO NOT MODIFY IT AND USE "platformio.ini":
+// https://docs.platformio.org/page/projectconf/section_env_build.html#build-flags
+//
 {
-    "env": {
-        "myDefaultIncludePath": [
-            "${workspaceFolder}/**"
-        ]
-    },
     "configurations": [
         {
-            "name": "Linux",
-            "intelliSenseMode": "linux-gcc-arm",
-            "defines": [
-                "CMAKE_TOOLCHAIN_FILE=${workspaceFolder}/arm-gcc-toolchain.cmake",
-                "CORE_CM4",
-                "CURRENT_BOARD=2",
-                "STM32WLE5xx"
-            ],
+            "name": "PlatformIO",
             "includePath": [
-                "${myDefaultIncludePath}",
-                "/opt/st/stm32cubeide_1.11.2/plugins/com.st.stm32cube.ide.mcu.externaltools.gnu-tools-for-stm32.10.3-2021.10.linux64_1.0.100.202210260954/tools/arm-none-eabi/include"
+                "C:/Users/kimball/code/blues/accelerator_dev/AQM/include",
```

Comment:
I don’t know a ton about this file, but I feel like it shouldn’t have a bunch of paths to files on your local machine.

Reply:
Ah it looks like this was supposed to be gitignored, so you should be safe to just delete the file from source control.